widgets/src/qsharp_widgets/__init__.py

```diff
@@ -35,3 +36,46 @@ class EstimateDetails(anywidget.AnyWidget):
     def __init__(self, estimates):
         super().__init__()
         self.estimates = estimates
+
+
+class Histogram(anywidget.AnyWidget):
```

Comment:
Does this do anything with DumpMachine/Message output? If not, I think it should just `print()` the output, just like we do in the original `run()` implementation.

Reply:
The intent is that eventually you'd be able to filter and navigate shots details by clicking on the histogram, same as playground does today. I really don't like the 'print by default'. If I'm running 10,000 shots and each has a handful of message and state events, I don't want 50,000 prints in the output by default.

Reply:
If we eventually intend to fold these into the UI then I think that's alright. (As long as we still print() in non-callback mode of course)